incubator/hnc/internal/reconcilers/object.go

```diff
@@ -660,13 +659,16 @@ func (r *ObjectReconciler) syncPropagation(ctx context.Context, log logr.Logger,
 func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst *unstructured.Unstructured, dst string) bool {
 	selector, err := selectors.GetSelector(inst)
 	if err != nil {
-		log.Error(err, "Invalid value")
+		log.Error(err, "Cannot propagate")
 	}
-	treeSelector, err := selectors.GetTreeSelector(inst)
-	if err != nil {
-		log.Error(err, "Invalid value")
+	treeSelector, err2 := selectors.GetTreeSelector(inst)
+	if err2 != nil {
+		log.Error(err2, "Cannot propagate")
+	}
```

Comment:
If you return early, you don't need to keep multiple `err` vars around.

Reply:
updated to more descriptive err names